src/magic_folder/cli.py

```diff
@@ -667,6 +668,10 @@ def __attrs_post_init__(self):
             ),
             Agent(self.reactor),
         )
+
+        if self.webport.isdigit():
+            self.webport = "tcp:{}".format(self.webport)
```

Comment:
Apart from the discussion on the linked ticket, the implementation would be better if instead constructed a `TCP4ServerEndpoint` here instead of doing string mashing.